.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+name: Netengine CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Python==${{ matrix.python-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
```

Comment:
I think we should test also `3.8`.
On `3.9` we have an issue in openwisp-radius, so it may be good to test that as well.
Python 3.6 will be EOL by the end of this year.